ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -236,12 +236,33 @@ spec:
           workspace: results
           subPath: summary
 
+    - name: resolve-pr-type
+      taskRef:
+        name: resolve-pr-type
+        kind: Task
+      runAfter:
+        - read-config
+      params:
+        - name: bundle_path
+          value: "$(tasks.detect-changes.results.bundle_path)"
+        - name: affected_catalogs
+          value: "$(tasks.detect-changes.results.affected_catalogs)"
+        - name: affected_catalog_operators
+          value: "$(tasks.detect-changes.results.affected_catalog_operators)"
+        - name: fbc-enabled
+          value: "$(tasks.read-config.results.fbc-enabled)"
```

Comment:
It looks like `read-config` can't handle PRs with catalog changes only. It only depends on bundle path that is empty. We will need to update the `read-config` task to also accept `affected_catalog_operators` result.

We can do it as part of this PR or create a separate ticket for it.

Reply:
I created https://issues.redhat.com/browse/ISV-5115 to fix this.

Reply:
Not sure what you mean: read-config just reads out info from ci.yaml and it locates the file based on operator_path which should be set even when the change only affects the catalog. From the tests I've done so far it seems to be working just fine.

Reply:
Based on my testing when a user submits just a catalog change the detect changes task doesn't return a value that is later consumed by the read-config tasks and read-config then returns default values (fbc = false).

You can see it here: https://github.com/redhat-openshift-ecosystem/operator-pipelines-test/pull/1627

I am going to take care of it in the ticket I linked above.